Week01/Problem03/wyou_03.py

```diff
@@ -0,0 +1,13 @@
+n = 600851475143
+factor = 2 
+
+
+primes= []
+while n>1:
```

Comment:
Interesting approach.  Might want to write some code comments describing what you're doing & how you're doing them, in future scripts, so your code becomes more readable.  Otherwise, it can take a long time to figure out what your plan/intentions are--especially if someone else (or you!) is trying to debug your code later.